GdxRuntimeException when restarting application

When *restarting* the application, for example after using the **EXIT GAME** button in the Menu screen, the application crashes due to a GdxRuntimeException: 

```
E/AndroidRuntime: FATAL EXCEPTION: GLThread 76234
    Process: no.ntnu.beardblaster, PID: 23105
    com.badlogic.gdx.utils.GdxRuntimeException: com.badlogic.gdx.utils.GdxRuntimeException: Cannot run tasks on an executor that has been shutdown (disposed)
        at com.badlogic.gdx.assets.AssetManager.handleTaskError(AssetManager.java:634)
        at com.badlogic.gdx.assets.AssetManager.update(AssetManager.java:413)
        at no.ntnu.beardblaster.view.LoadingScreen.update(LoadingScreen.kt:35)
        at no.ntnu.beardblaster.view.LoadingScreen.render(LoadingScreen.kt:44)
        at ktx.app.KtxGame.render(game.kt:55)
        at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:471)
        at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1592)
        at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1285)
     Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Cannot run tasks on an executor that has been shutdown (disposed)
        at com.badlogic.gdx.utils.async.AsyncExecutor.submit(AsyncExecutor.java:59)
        at com.badlogic.gdx.assets.AssetLoadingTask.handleAsyncLoader(AssetLoadingTask.java:114)
        at com.badlogic.gdx.assets.AssetLoadingTask.update(AssetLoadingTask.java:91)
        at com.badlogic.gdx.assets.AssetManager.updateTask(AssetManager.java:561)
        at com.badlogic.gdx.assets.AssetManager.update(AssetManager.java:411)
        at no.ntnu.beardblaster.view.LoadingScreen.update(LoadingScreen.kt:35) 
        at no.ntnu.beardblaster.view.LoadingScreen.render(LoadingScreen.kt:44) 
        at ktx.app.KtxGame.render(game.kt:55) 
        at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:471) 
        at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1592) 
        at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1285) 
E/AndroidGraphics: waiting for pause synchronization took too long; assuming deadlock and killing
I/Process: Sending signal. PID: 23105 SIG: 9
```

--- a/core/src/main/kotlin/no/ntnu/beardblaster/BeardBlasterGame.kt
+++ b/core/src/main/kotlin/no/ntnu/beardblaster/BeardBlasterGame.kt
@@ -2,6 +2,7 @@ package no.ntnu.beardblaster
 
 import com.badlogic.gdx.Application
 import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.assets.AssetManager
 import com.badlogic.gdx.graphics.OrthographicCamera
 import com.badlogic.gdx.graphics.g2d.Batch
 import com.badlogic.gdx.graphics.g2d.SpriteBatch
@@ -26,7 +27,7 @@ class BeardBlasterGame : KtxGame<AbstractScreen>() {
         // Set debug level
         Gdx.app.logLevel = Application.LOG_DEBUG
         LOG.debug { "Create game instance" }
-        
+        Assets.assetManager = AssetManager()
         initScreens()
     }
 

--- a/core/src/main/kotlin/no/ntnu/beardblaster/assets/Assets.kt
+++ b/core/src/main/kotlin/no/ntnu/beardblaster/assets/Assets.kt
@@ -10,7 +10,7 @@ import com.badlogic.gdx.graphics.g2d.TextureAtlas
 object Assets {
 
     // Define all assets that must be loaded during loading screen
-    var assetManager = AssetManager()
+    lateinit var assetManager: AssetManager
     val loadBar = AssetDescriptor<Texture>("graphics/load_bar.png", Texture::class.java)
     val loginImg = AssetDescriptor<Texture>("graphics/login.png", Texture::class.java)
     val test1 = AssetDescriptor<Texture>("bar_container.png", Texture::class.java)

